### PR TITLE
search working, answers correctly populating, extension removal

### PR DESCRIPTION
**Pull request by murkymode**

### CODE REVIEW

File: client/src/components/Q&A/AnswerRow.jsx [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
-      className="answersRow"
+      className="answers-row"
       style={{ border: '2px solid slateblue' }}
     >
       A:
-      {/* {questions.map((question) => question.answers.body)} */}
-      {tempQuestion.answers[tempID].body}
+      {answers.map((answer) => answer.body)}
+      <br />
+      {answers.length > 1 ? <span>Load More Answers</span> : <></>}
```
Comment (alishaedington) [inline client/src/components/Q&A/AnswerRow.jsx]: just keep in mind that it should load 2 answers by default if there are two available. so here maybe answers.length > 2.

Comment (murkymode) [reply client/src/components/Q&A/AnswerRow.jsx]: Nice catch! I'll change that and resubmit

File: client/src/components/Q&A/SearchBar.jsx [hunk 2]
```diff
@@ -1,17 +1,27 @@
 /* eslint-disable import/extensions */
 import React from 'react';
 
-export default function SearchBar() {
+export default function SearchBar({ setSearchTerm }) {
+  function handleSearch(event) {
+    const search = event.nativeEvent.target.value;
```
Comment (alishaedington) [inline client/src/components/Q&A/SearchBar.jsx]: does event.target.value not work here?

File: client/src/components/Q&A/SearchBar.jsx [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
-      className="searchBar"
+      className="search-bar"
       style={{ border: '2px solid slateblue' }}
     >
       Hello from Search Bar
       <form>
         <input
-          className="searchInput"
+          className="search-input"
           placeholder="Have a question? Search for answers…"
```
Comment (alishaedington) [inline client/src/components/Q&A/SearchBar.jsx:23]: Does the search bar look for answers or questions? Or both?

**Review by alishaedington — DISMISSED** [review]
looks solid. comments can be taken care of whenever.

File: client/src/components/Q&A/SearchBar.jsx (hunk 2 above)
Comment (murkymode) [reply client/src/components/Q&A/SearchBar.jsx]: Looks like it does! Switching that now

File: client/src/components/Q&A/SearchBar.jsx (hunk 3 above)
Comment (murkymode) [reply client/src/components/Q&A/SearchBar.jsx:23]: I think it filters the questions, but I could be mistaken too. In the business req's it doesn't mention searching by answers.

**Review by alishaedington — APPROVED** [review]
looks good.

**Pull request merged**